lib/routes/misskey/home-timeline.ts

```diff
@@ -0,0 +1,101 @@
+import { config } from '@/config';
+import ConfigNotFoundError from '@/errors/types/config-not-found';
+import { Route, ViewType } from '@/types';
+import got from '@/utils/got';
+import { queryToBoolean } from '@/utils/readable-social';
+import querystring from 'querystring';
+import utils from './utils';
+
+export const route: Route = {
+    path: '/timeline/home/:site/:routeParams?',
+    categories: ['social-media', 'popular'],
```

Comment:
New route won't be popular
```suggestion
    categories: ['social-media'],
```